src/NorthwindWebsite/Core/Constants/HttpContentConstants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NorthwindWebsite.Core.Constants
+{
+    public static class HttpContentConstants
```

Comment:
No sure that this class is needed:
1. You can move `ImagesControllerRoute` to `ImageMiddlewareConfiguration` because it has only single place of usage.
2. You can move `ImageBmp` to `ImageCachingMiddleware` because it has only single place of usage.